detect/app.go

```diff
@@ -60,6 +63,16 @@ func GetAppNameFromGitRemote(directory string, remoteName string) (string, error
 	return "", errgo.Newf("[detect] Scalingo Git remote hasn't been found")
 }
 
+func extractAppNameFromCommandLine(c *cli.Context) string {
+	for _, context := range c.Lineage() {
```

Comment:
`context` is not a very good variable name as it is often the name of an imported package. Suggestion:

```suggestion
	for _, cliContext := range c.Lineage() {
```